.github/CODEOWNERS

```diff
@@ -74,7 +74,7 @@ clang/tools/clang-offload-*/ @intel/dpcpp-tools-reviewers
 ESIMD/ @intel/dpcpp-esimd-reviewers
 esimd/ @intel/dpcpp-esimd-reviewers
 sycl/include/sycl/ext/intel/esimd.hpp @intel/dpcpp-esimd-reviewers
-sycl/doc/extensions/experimental/sycl_ext_intel_esimd/ @intel/dpcpp-esimd-reviewers
+sycl/doc/extensions/supported/sycl_ext_intel_esimd/ @intel/dpcpp-esimd-reviewers
```

Comment:
```suggestion
sycl/doc/extensions/**/sycl_ext_intel_esimd/ @intel/dpcpp-esimd-reviewers
```

Reply:
@bader - What is the idea of the additional fix? 
The old 'sycl_ext_intel_esimd' folder gets completely removed from 'experimental/*' by this change.

Reply:
It will make our life easier if we later move this doc to other `extensions` sub-directories (e.g. supported -> deprecated -> removed).

Reply:
> It will make our life easier if we later move this doc to other `extensions` sub-directories (e.g. supported -> deprecated -> removed).

(supported->deprecated->removed)?! This does not sound optimistic. I'll do the change though.

Reply:
It's pragmatic. :)